lib/java/gradle.properties

```diff
@@ -0,0 +1,35 @@
+# This file is shared currently between this Gradle build and the
+# Ant builds for fd303 and JavaScript. Keep the dotted notation for
+# the properties to minimize the changes in the dependencies.
+thrift.version=1.0.0
+thrift.groupid=org.apache.thrift
+release=false
+
+# Local Install paths
+install.path=/usr/local/lib
+install.javadoc.path=/usr/local/lib
+
+# Test execution properties
+testPort=9090
+
+# Maven dependency download locations
+mvn.repo=http://repo1.maven.org/maven2
+apache.repo=https://repository.apache.org/content/repositories/releases
+
+# Apache Maven publish
+license=http://www.apache.org/licenses/LICENSE-2.0.txt
+maven-repository-url=https://repository.apache.org/service/local/staging/deploy/maven2
+maven-repository-id=apache.releases.https
+
+# Ant Maven Task Jar Information, not used by this project anymore
+mvn.ant.task.version=2.1.3
+mvn.ant.task.url=${mvn.repo}/org/apache/maven/maven-ant-tasks/${mvn.ant.task.version}
+mvn.ant.task.jar=maven-ant-tasks-${mvn.ant.task.version}.jar
+
+# Dependency versions
+httpclient.version=4.4.1
```

Comment:
Are these hard or soft dependencies?  There are newer versions of these.

Reply:
One hurdle at a time! I did not want to open that can of worms yet. Perhaps once we have this building and passing cross-tests we can revisit refreshing dependencies. There are some deprecation warnings in the code that I can see and it is possible they should be addressed.